src/plugins/embeddable/public/react_embeddable_system/react_embeddable_renderer.tsx

```diff
@@ -69,6 +69,7 @@ export const ReactEmbeddableRenderer = <
     | 'hideLoader'
```

Comment:
I know it's not very helpful without repro steps, but I found myself in this state while doing some multiple edits and cannot reproduce it. Basically it looks like visualization state was wiped out and we got a very weird empty state. 
<img width="581" alt="Screenshot 2024-11-22 at 15 04 23" src="https://github.com/user-attachments/assets/6d8d5481-1454-458f-a750-87efca48de19">

[export.json](https://github.com/user-attachments/files/17871220/export.json)

I will try to get some stable repro, but in the meantime, another problem with it is that `Edit Lens` link doesn't work at all (doesn't trigger any error either)

Reply:
I think it's a false alarm and I probably just removed the linked vis from library. The behavior on Main is similar so all good. I hope 😅

Reply:
Me too and same problem: can't replicate it

Reply:
Ok, that's more worrying 😭 I'll be clicking more on Monday 🤞🏼